src/FSharpSource.targets

```diff
@@ -8,74 +8,43 @@
 
   <Import Project="..\build\targets\GitHash.props" />
 
-  <Choose>
-    <When Condition="'$(Configuration)' == 'Debug' or '$(Configuration)' == 'Release' ">
-      <PropertyGroup>
-        <SkipSigning>false</SkipSigning><!-- Skip using the ,net toolset to generate signing . -->
-        <UseOpenSourceSign>true</UseOpenSourceSign>
-      </PropertyGroup>
-      <Choose>
-        <When Condition="'$(ProjectLanguage)' == 'FSharp'">
-          <Choose>
-            <!-- Some unit test assemblies must not have strong names since they reference other assemblies that don't have strong names -->
-            <!-- These unit test assemblies can't use InternalsVisibleTo into assemblies with strong names. -->
-            <When Condition="'$(StrongNames)' == 'false'" >
-              <PropertyGroup>
-                <SkipSigning>true</SkipSigning>
-                <UseOpenSourceSign>false</UseOpenSourceSign>
-                <MicroBuildAssemblyFileLanguage>fs</MicroBuildAssemblyFileLanguage>
-              </PropertyGroup>
-            </When>
-            <!-- In the Microsoft build we just delay-sign everything with the MSFT key -->
-            <!-- We have to do unit test DLLs well because they can see the internals of other strong-named DLLs -->
-            <Otherwise>
-              <PropertyGroup>
-                <DelaySign>true</DelaySign>
-                <StrongNames>true</StrongNames>
-                <MicroBuildAssemblyFileLanguage>fs</MicroBuildAssemblyFileLanguage>
-              </PropertyGroup>
-            </Otherwise>
-          </Choose>
-        </When>
-        <Otherwise>
-          <!-- Not F# code -->
-          <PropertyGroup>
-            <SignAssembly>true</SignAssembly>
-            <AssemblyOriginatorKeyFile>$(FSharpSourcesRoot)\fsharp\msft.pubkey</AssemblyOriginatorKeyFile>
-            <StrongNames>true</StrongNames>
-            <DelaySign>true</DelaySign>
-            <MicroBuildAssemblyFileLanguage>fs</MicroBuildAssemblyFileLanguage>
-          </PropertyGroup>
-        </Otherwise>
-      </Choose>
-    </When>
-    <Otherwise>
-      <PropertyGroup Condition="'$(StrongNames)' != 'true'">
-        <!-- For the proto build we don't use strong names. -->
-        <DefineConstants>NO_STRONG_NAMES;$(DefineConstants)</DefineConstants>
-        <MicroBuildAssemblyFileLanguage>fs</MicroBuildAssemblyFileLanguage>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+  <PropertyGroup>
```

Comment:
@dsyme I'm not clear how the signing settings end up for a proto build with this new factoring, I.e ... when Configuration='proto'

Reply:
It uses the default settings used for all compiler binaries - delay signed on windows, test.snk signed on Mono

Reply:
The default settings for non-Mono are in FSharpSource.Settings.targets.  I'm not sure why, they were there before
```
  <PropertyGroup>
    <SkipSigning>false</SkipSigning>
    <UseOpenSourceSign>true</UseOpenSourceSign>
    <SignAssembly>true</SignAssembly>
    <AssemblyOriginatorKeyFile>$(FSharpSourcesRoot)\fsharp\msft.pubkey</AssemblyOriginatorKeyFile>
    <StrongNames>true</StrongNames>
    <DelaySign>true</DelaySign>
  </PropertyGroup>

```